tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "experimentalDecorators": true,
     "moduleResolution": "node",
     "strictNullChecks": true, // enable strict null checks as a best practice
-    "module": "ES2015", // specify module code generation
+    "module": "commonjs", // specify module code generation
```

Comment:
Curious as to why this change is needed. @qiweiii .

Reply:
It's because ts-node can only run our scripts when using 'commonjs', it's caused by a bug in ts-node(or maybe node): https://github.com/TypeStrong/ts-node/issues/935, and here is a related issue: https://github.com/TypeStrong/ts-node/issues/1007

Reply:
This might effect our build step.

Reply:
It will make our exported modules non-treeshakeable I believe.

Reply:
Never mind. This is good